src/bridge/PublicitySyncer.ts

```diff
@@ -23,47 +24,44 @@ export class PublicitySyncer {
         mappings: {
             [roomId: string]: string[];
         };
-        networkToRooms: {
-            [networkId: string]: string[];
-        };
         channelIsSecret: {
-            [networkId: string]: boolean;
+            [networkIdChannel: string]: boolean;
             // '$networkId $channel': true | false
         };
         roomVisibilities: {
             [roomId: string]: "private"|"public";
         };
     } = {
         mappings: {},
-        networkToRooms: {},
         channelIsSecret: {},
         roomVisibilities: {},
     };
-    constructor (private ircBridge: IrcBridge) { }
-
+    private initModeQueue: Queue<{server: IrcServer; channel: string}>;
+    constructor (private ircBridge: IrcBridge) {
+        this.initModeQueue = new Queue(this.initModeForChannel.bind(this));
+    }
 
-    public async initModeForChannel(server: IrcServer, chan: string) {
+    public async initModeForChannel(opts: {server: IrcServer; channel: string}) {
```

Comment:
A semi-colon is valid here? Is it equivalent to a comma?
```suggestion
    public async initModeForChannel(opts: {server: IrcServer, channel: string}) {
```

Reply:
Yup. Apparently our linter favours it.